src/client/auth/oidc.rs

```diff
@@ -177,6 +160,23 @@ pub struct IdpServerResponse {
     pub refresh_token: Option<String>,
 }
 
+/// Constructs the first client message in the OIDC handshake for speculative authentication
```

Comment:
Is it ok for this to return an empty speculative command when there is no access token, or should we actually make this return an Option<Command>?